platform/engineering/code_review_guidelines.md

```diff
@@ -4,41 +4,58 @@ This document covers rules for engaging the VSP team in [pull request reviews](h
-      * **Do not post in Slack channels requesting VSP code reviews** unless it has been more than 24 hours since requesting review on GitHub.
-    * After sending your code for review, don't add additional changes! Code needs to be stable for a safe review.
-    * If non-trivial changes to your code are made *after approval by VSP team* and submitted without a re-review, your change is subject to being reverted.
-
-* **First review**: 
-  * Use a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
-  * Tag your PR with your team-specific label(s)
-  * For this first review, your code should be reviewed and approved by a member of your team. 
-  * First review should cover:
-    * **Correctness** of the code (does it correctly implement the described feature?)
-    * **Code quality**: is it readable? Is it language-idiomatic?
-    * **Does it have screenshots of the change?** (FE changes)
-
-* **Description**: make it clear what the change is describing.
-    * **PR title** should be a concise summary of the change.
-    * In your **PR description**, discuss how code changes are related to the pull request objectives.
-    * Reference the **GitHub issues** your PR relates to, in order to provide reviewers more context.
-
-* **Size/complexity**: keep your changes scoped to one feature at a time. This means:
-    * **Size**: changes should be smaller than **300** changed lines (not including tests or whitespace). *If your change can't be smaller than this, talk to the team in #vfs-platform-support before asking for a review.*
+We expect that most developers have worked with GitHub and review processes before, but want to be clear about our norms to reduce confusion and help code flow through the process as quickly and reliably as possible.
+
+
+## The pull request review lifecycle in brief
+
+1. For initial review by your project team, create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/)
+2. Once you are ready for the VSP team to review your PR, use the GitHub *Ready for review* button in the merge section of your pull request. An engineer should review the code within 2 days. 
+3. When all review comments have been resolved, the PR can be merged into the master branch for deployment.
+
+*Hint*: Smaller, focused pull requests will make this easier for everyone!
+
+## Code Review Guidelines
+
+Your code will be automatically reviewed by automated systems for format, common errors, typical
+language idioms, and code preferences using language-standard tools. 
+
+When your code is reviewed, here are some of the things we're looking for:
+
+* *Correctness*. Does the code correctly implement the described feature?
+* *Code quality*. Is the code readable? Is the code language-idiomatic?
+* For front end changes, are screenshots of the change(s) included?
+
+
+
+## Helpful hints
```

Comment:
Can we pick a different title? This doc should be *requirements* we're imposing on teams sending things to us, not *advice*, and headings/wording should reflect that.

Reply:
Done.